### PR TITLE
Admins can respawn

### PR DESCRIPTION
**Pull request by Rohesie**
Makes testing easier, this doesn't really affect the servers.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Some poor downstream is gonna get fucked up by this

**Comment by ATH1909** [comment]
> Some poor downstream is gonna get fucked up by this

fulp's gonna have a !!FUN!! time

**Comment by Jared-Fogle** [comment]
I don't know if I'm a fan of diverging from the main config too much, even when the config used in the repo is just used for testing. I'd think that a separate "great for testing" config might be interesting, but it'd also be a pain to maintain.

**Comment by Ryll-Ryll** [comment]
Maybe have it depend on defining TESTING?

**Comment by Cyberboss** [comment]
I don't think this is a good idea for three reasons.

- This is diverging from the live config
- Git clones have admin powers on localhost by default and can respawn themselves with the PP
- Real servers should know what they're doing

**Comment by Rohesie** [comment]
Config restored, gave the option to those with the `R_SPAWN` perms to respawn if they click on the prompt.
This is logged already for live games, and nothing they couldn't do through the PP already.

**Comment by Cyberboss** [comment]
`R_SPAWN` is for atom creation. I'd argue re-spawning is 100% a moderation thing

**Comment by PKPenguin321** [comment]
Check for whatever gives you the power to Send to Lobby

**Comment by Rohesie** [comment]
Currently it's `R_ADMIN`
I can revert it to it, sure.

**Comment by Jared-Fogle** [comment]
Much more in favor of this instead of updating the config. :+1:

**Review by Cyberboss — APPROVED** [review]

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -629,8 +629,9 @@
 	set name = "Respawn"
 	set category = "OOC"
 
-	if (CONFIG_GET(flag/norespawn))
+	if (CONFIG_GET(flag/norespawn) && (!check_rights_for(usr.client, R_ADMIN) || alert(usr, "Respawn configs disabled. Do you want to use your permissions to circumvent it?", "Respawn", "Yes", "No") != "Yes"))
```
Comment (Dennok) [inline code/modules/mob/mob.dm:632]: Respawn configs disabled. Do you want to use your badmintry to overwhelm it?

**Pull request merged**